src/help-widget.ts

```diff
@@ -205,4 +212,30 @@ export class BBHelpHelpWidget {
   private isCollapsed() {
     return this.container.classList.contains(HELP_CLOSED_CLASS);
   }
+
+  private watchWindowWidth() {
+    window.addEventListener('resize', () => {
+      this.resizeContainer();
+    });
+  }
+
+  private resizeContainer() {
+    // width pulled from $screenXsMax scss variable
+    if (window.innerWidth < 479) {
```

Comment:
Make sure this is the same value we are tracking against in the current `bb-help` project.  We should also set this as a constant variable at the top of this file.  `const SCREEN_XS_MAX: number = 479`

Reply:
I'm not sure where to find this value, exactly. Is this coming from bootstrap? Skyux? Not sure.

Reply:
I think the the boostrap or skyux 1 project
https://github.com/blackbaud/bb-help/blob/master/assets/scss/app.scss

Reply:
https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss 
This seems to indicate that 479 is the right size

Reply:
https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L320

I think that says 767.  

$screen-sm:                  768px !default;
$screen-sm-min:              $screen-sm !default;
$screen-xs-max:              ($screen-sm-min - 1) !default;

In addition we should make this a variable that's easy to read/see.  no need for the comment.

Reply:
```
// Extra small screen / phone
//** Deprecated `$screen-xs` as of v3.0.1
$screen-xs:                  480px !default;
//** Deprecated `$screen-xs-min` as of v3.2.0
$screen-xs-min:              $screen-xs !default;
//** Deprecated `$screen-phone` as of v3.0.1
$screen-phone:               $screen-xs-min !default;
...
$screen-xs-max:              ($screen-sm-min - 1) !default;
```

I think this is where the 479 number is coming from specifically, could we confirm which is right before moving forward? 

This is the number that's showing up on master in the Media Query Inspector in chrome: 
![image](https://user-images.githubusercontent.com/38473653/44724017-97512680-aa9f-11e8-953d-839f6ef6fde7.png)

Just want to make sure we are using the right number, since 767 seems really big for mobile.

Reply:
https://docs.blackbaud.com/bb-help-docs/

Inspect and shrink the page here.  The mobile sizing on the current widget pops at 767.

Where did you find the above reference?

Reply:
Cool that's what I needed to know. The code I posted above was from the bootstrap styles that I posted yesterday, but if we are overwriting that then it doesn't matter